.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,22 @@
+name: Deploy
+
+on:
+  push:
+    branches: ['feat/**']
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+    
+      - name: create env file
+        run: |
+          touch .env
+          echo DEPLOY_PRIVATE_KEY=${{ secrets.DEPLOY_PRIVATE_KEY }} >> .env          
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#env

```yaml
env:
  DEPLOY_PRIVATE_KEY=${{ secrets.DEPLOY_PRIVATE_KEY }}
```